ignore-words.txt

```diff
@@ -0,0 +1 @@
+crate
```

Comment:
What is this file for and why do we need it?

Reply:
Removed. Left there previously when importing additional libraries for testing

Reply:
Update: Sorry they are actually needed because tox spellcheck thinks crate is supposed to be spelled create. Added back